### DISCUSSION TITLE
Kudos to whoever designed the BB Gradients!

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#5615] [Show and tell]
Really beautiful gradients guys, whoever came up with them and the names.

**Comment by melohagan** [comment]
Hey @poirazis 

Thank you very much for the kind feedback! :)

I've added a section to the docs so users can see the options available to them: https://docs.budibase.com/docs/container#container-background-color

**Reply by poirazis** [reply]
Here is a show of of what i used them for :) 
<img width="1468" alt="Screenshot 2022-04-28 at 12 29 21 PM" src="https://user-images.githubusercontent.com/80512170/165722776-c54f25ad-06b4-43c1-95a0-f706c7d7def2.png">

**Reply by melohagan** [reply]
That looks really nice :)
Really like the style of your buttons as well!

**Reply by poirazis** [reply]
When it's done, you can use it as a template if you wish to give it to the world !

**Reply by melohagan** [reply]
Definitely, feel free to share the app export when you're done!
